pkgs/applications/networking/cluster/nerdctl/default.nix

```diff
@@ -29,16 +37,35 @@ buildGoModule rec {
   # Many checks require a containerd socket and running nerdctl after it's built
   doCheck = false;
 
-  postInstall = ''
-    wrapProgram $out/bin/nerdctl \
-      --prefix PATH : "${lib.makeBinPath ([ buildkit ] ++ extraPackages)}" \
-      --prefix CNI_PATH : "${cni-plugins}/bin"
+  postInstall =
+    let
+      binPath =
+        lib.makeBinPath ([
+          libselinux
+          rootlesskit
+          containerd
+          runc
+          slirp4netns
+          util-linux
+          iptables
+          iproute2
+        ]);
+    in
+    ''
+      wrapProgram $out/bin/nerdctl \
+        --prefix PATH : "${lib.makeBinPath ([ buildkit runc ] ++ extraPackages)}" \
+        --prefix CNI_PATH : "${cni-plugins}/bin"
 
-    installShellCompletion --cmd nerdctl \
-      --bash <($out/bin/nerdctl completion bash) \
-      --fish <($out/bin/nerdctl completion fish) \
-      --zsh <($out/bin/nerdctl completion zsh)
-  '';
+      install -D extras/rootless/containerd-rootless.sh $out/bin/containerd-rootless
```

Comment:
Any reason to drop `.sh` suffix?

Reply:
I suggest not to drop it, for consistency with the upstream

Reply:
could leave it but when a script is packaged in `xyz/bin` typically you drop the suffixes
If it was just local or hidden the prefix would stay